index.html

```diff
@@ -5,22 +5,87 @@
+      <p class="grid-item">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul><li><button class="grid-item" type="button">html</button></li><li><button class="grid-item" type="button">CSS</button></li><li><button class="grid-item" type="button">Javascript</button></li></ul>
+      <button type="button" class="project">See Project</button>
+    </div>
+    <div class="grid-container">
+      <div class="grid-item"><img alt="twitter" src="Snapshoot_Portfolio_1.png"></div>
+      <div class="grid-item"><h2>Multi-Post Stories</h2></div>
+      <div class="grid-item"><ul><li class="b"><b>CANOPY</b></li><li class="a">Back End Dev</li> <li class="a">2015</li></ul></div>
+      <p class="grid-item">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul><li><button class="grid-item" type="button">html</button></li><li><button class="grid-item" type="button">CSS</button></li><li><button class="grid-item" type="button">Javascript</button></li></ul>
+      <button type="button" class="project">See Project</button>
+    </div>
+    <div class="grid-container">
+      <div class="grid-item"><img alt="twitter" src="Snapshoot_Portfolio_2.png"></div>
+      <div class="grid-item"><h2>Tonic</h2></div>
+      <div class="grid-item"><ul><li class="b"><b>CANOPY</b></li><li class="a">Back End Dev</li> <li class="a">2015</li></ul></div>
+      <p class="grid-item">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul><li><button class="grid-item" type="button">html</button></li><li><button class="grid-item" type="button">CSS</button></li><li><button class="grid-item" type="button">Javascript</button></li></ul>
+      <button type="button" class="project">See Project</button>
     </div>
-
+    <div class="grid-container">
+      <div class="grid-item"><img alt="twitter" src="Snapshoot_Portfolio_3.png"></div>
+      <div class="grid-item"><h2>Multi-Post Stories</h2></div>
+      <div class="grid-item"><ul><li class="b"><b>CANOPY</b></li><li class="a">Back End Dev</li> <li class="a">2015</li></ul></div>
+      <p class="grid-item">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul><li><button class="grid-item" type="button">html</button></li><li><button class="grid-item" type="button">CSS</button></li><li><button class="grid-item" type="button">Javascript</button></li></ul>
+      <button type="button" class="project">See Project</button>
+    </div>
+    <div class="grid-container">
+    <h1 class="grid-item box2">About Myself<br></h1>
+    <p class="grid-item">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div class="grid-item"><a href="#">LET'S CONNECT</a></div>
```

Comment:
Kindly make sure you use the same font as figma's to be as close to the design as you can 📝
#### Your design 
![Captura](https://user-images.githubusercontent.com/15807343/149247090-c55bba96-c6c1-4785-84c5-f9a309287a83.PNG)

#### Figma's design 
![Captura2](https://user-images.githubusercontent.com/15807343/149247104-c2e38714-fe09-4593-9e2d-31d29d750602.PNG)